web/src/dialog_widget.js

```diff
@@ -118,6 +119,29 @@ export function launch(conf) {
 
     const $submit_button = $dialog.find(".dialog_submit_button");
 
+    if (conf.update_submit_on_state_change) {
+        const originalValues = {};
+        const currentValues = {};
+        $(() => {
+            $submit_button.prop("disabled", true);
+            $(".modal__content :input").each(function () {
+                originalValues[$(this).attr("name")] = currentValues[$(this).attr("name")] =
```

Comment:
I think it would be much more readable if we use a variable like `property_name` for `$(this).attr("name")` and also set the values for the two objects in separate lines.

Reply:
> I think it would be much more readable if we use a variable like `property_name` for `$(this).attr("name")` and also set the values for the two objects in separate lines.

can we use `field_name` and `field_value`  or `input_name` and `input_value` for `$(this).attr("name")` and `$(this).val()`?